content/operator/euskotren/index.de.md

```diff
@@ -0,0 +1,91 @@
+---
+draft: false
+title: 'Euskotren'
+country:
+  - 'spain'
+  - 'france'
+operator: 'euskotren'
+---
+
+{{< wip >}}
+
+Im Norden Spaniens zwischen der französischen Grenze bei Hendaye und der Metropole Bilbao verbindet ein Netz der baskischen Eisenbahn die malerischen Küstenorte und das gebirgige Hinterland. Euskotren ist Eigentum der baskischen Regionalregierung.
+
+## Zusammenfassung
+
+- Vor Ort wird bei Vorlage des FIP Ausweises kostenfrei ein Ticket ausgestellt, das 7 Tage gültig ist und zur Nutzung des gesamten Euskotren Netzes mit Ausnahme von Bussen berechtigt.[^1]
+- Mit dem Ticket können die Züge, die U-Bahnlinie L3 in Bilbao sowie Straßenbahnen und die Standseilbahn "Funicular de Larreineta" genutzt werden.
+- Das Ticket kann nach Ablauf auch mehrfach pro Jahr neu ausgestellt werden. Die sonst übliche Limitierung bei FIP Freifahrtscheinen gilt bei Euskotren nicht.
+
+## Gültigkeit FIP Tickets
+
+FIP Freifahrtsschein: ✅, kann vor Ort ausgestellt werden.
+
+## Zugkategorien und Reservierungen
+
+{{% expander "Meterspur-Bahn" traincategory "category" %}}
+**Beschreibung:**
+
+Euskotren bereit Züge auf der elektrifizierten Meterspur-Bahn in den Provinzen Bizkaia und Gipuzkoa. Der ausgestellte FIP Freifahrtsschein wird auf allen Linien, einschließlich der Linie bis zum TGV-Bahnhof Hendaye in Frankreich, anerkannt.
+{{% /expander %}}
+
+{{% expander "Straßenbahnen" traincategory "category" %}}
+**Beschreibung:**
+
+Der ausgestellte FIP Freifahrtsschein wird sowohl bei der Straßenbahn in Bilbao als auch in Vitoria-Gasteiz anerkannt.
+{{% /expander %}}
+
+{{% expander "U-Bahnlinie L3 in Bilbao ℹ️" traincategory "category" %}}
+**Beschreibung:**
+
+In Bilbao steht ein U-Bahn-Netz mit drei Linien zur Verfügung. Auf der Linie L3 wird der FIP Freifahrtsschein anerkannt.
+
+ℹ️ Auf allen anderen Linien (L1 und L2) im Metronetz von Bilbao wird FIP nicht anerkannt. Nur die Linie L3 wird von Euskotren betrieben.[^3]
+{{% /expander %}}
+
+{{% expander "Standseilbahn \"Funicular de Larreineta\"" traincategory "category" %}}
+**Beschreibung:**
+
+Die historische Standseilbahn zwischen Trapagaran und Larreineta wird von Euskotren betrieben und verbindet die Bergsiedlungen oberhalb von Trapagaran mit dem Tal.
+Für die Standseilbahn wird ein kostenfreies Ticket an der jeweiligen Station ausgestellt.
+{{% /expander %}}
+
+{{% expander "Regionalbuslinien ⛔" buscategory "category" %}}
+**Beschreibung:**
+
+FIP wird in den Bussen von Euskotren nicht anerkannt.
+{{% /expander %}}
```

Comment:
Busse schreiben wir immer in den besonderen Tarifbedingungen, dass dort FIP nicht gilt.